src/views/Swap/index.tsx

```diff
@@ -68,6 +70,32 @@ const Label = styled(Text)`
   color: ${({ theme }) => theme.colors.secondary};
 `
 
+const SwitchIconButton = styled(IconButton)`
+  box-shadow: inset 0px -2px 0px rgba(0, 0, 0, 0.1);
+  .icon-up-down {
+    display: none;
+  }
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.primary};
+    .icon-down {
+      display: none;
+      fill: white;
+    }
+    .icon-up-down {
+      display: block;
+      fill: white;
+    }
+  }
+`
+
+const ArrowUpDownIcon: React.FC<SvgProps> = (props) => {
```

Comment:
will move this to uikit https://github.com/pancakeswap/pancake-toolkit/pull/350/files